src/components/WeterestGrid/GridPost.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const GridPost = ({ pins, heights }) => {
+  return (
+    <div className="GridPost">
+      {pins.map((pin, index) => {
+        const offsetY = currentIndex => {
+          const columnCount = 5;
+          const remainder = currentIndex % columnCount;
+          let result = 0;
+
+          if (currentIndex < columnCount) {
+            return 0;
+          }
+
+          for (let i = 0; i <= currentIndex - columnCount; i++) {
+            if (i % columnCount === remainder) {
+              result = result + heights[i];
+            }
+          }
+
+          return result;
+        };
+
+        return (
+          <Post
+            key={pin.id}
+            style={{
+              height: `${heights[index]}px`,
+              transform: `translateX(${
+                (index % 5) * 252
+              }px) translateY(${offsetY(index)}px)`,
+            }}
+          >
+            <PostInner>
+              <Content>
+                <ContentInner>
+                  <IconsGroup>
+                    <Icons>
+                      <PinSave>Save</PinSave>
+                    </Icons>
+                    <IconsOverlay />
+                  </IconsGroup>
+                  <Image src={pin.image_url} />
+                  <ImageSkeleton
+                    style={{
+                      backgroundColor: `${pin.color}`,
+                      height: `${heights[index] - 24}px`,
+                    }}
+                  />
+                </ContentInner>
+              </Content>
+            </PostInner>
+          </Post>
+        );
+      })}
+    </div>
+  );
+};
+
+export default GridPost;
+
+const PostWidth = `252px`;
```

Comment:
해당 값이 `252` 다른 곳에서도 사용되고 있네요! 
위에 드렸던 리뷰처럼 '252'값을 아래와 같이 상수로 관리해도 좋을 것 같습니다.
```suggestion
const POST_WIDTH = 252;
```